src/selectors/index.js

```diff
@@ -19,6 +19,8 @@ import isCurrentProjectSyntacticallyValid
   from './isCurrentProjectSyntacticallyValid';
 import isExperimental from './isExperimental';
 import isGistExportInProgress from './isGistExportInProgress';
+import isRepoExportInProgress from './isRepoExportInProgress';
+import isClassroomExportInProgress from './isClassroomExportInProgress';
```

Comment:
I could go either way on having three separate selectors but I think it’s basically good. Selectors that take arguments are a little more complicated to deal with anyway.

Reply:
Yeah was going back and forth here. That sounds good to me!